puppetboard/static/js/environments.js

```diff
@@ -0,0 +1,11 @@
+(function () {
+  var $;
+
+  $ = jQuery;
+
+  $('#switch_env').change(function() {
+    path = location.pathname.split('/');
+    path[1] = $(this).find(':selected').text();
+    location.assign(path.join('/'))
```

Comment:
Same thing again. Semantic UI is perfectly mobile friendly though we're on a pretty old version as it currently stands. Feel free to change it around though but do stick to 1 framework for everything, otherwise it becomes unnecessarily complex.